src/_assets/scss/inc/showcase.scss

```diff
@@ -0,0 +1,89 @@
+.showcase-grid {
```

Comment:
Nit: Could we use BEM here? Or my version of BEM? 😅 Single underscore for nesting (never found a use case for the double underscore from the original proposal): `.showcase_grid`, `.showcase_item` or even `.showcase_grid_item`

Reply:
You know what? It doesn't matter to me.